src/App.tsx

```diff
@@ -31,22 +35,33 @@ export class App extends React.Component<{}, State> {
     const { count } = this.state;
 
     return (
-      <div className="App">
-        <h1 className="App__title">
-          {`Count: ${count}`}
-        </h1>
-
-        <button type="button" className="App__add-one" onClick={this.addOne}>
-          Add 1
-        </button>
-
-        <button type="button" className="App__add-100" onClick={this.add100}>
-          Add 100
-        </button>
-
-        <button type="button" className="App__increase" onClick={this.increase}>
-          Increase
-        </button>
+      <div className="wrapper">
+        <div className="App box">
+          <h1 className="App__title title">
+            {`Count: ${count}`}
+          </h1>
+          <button
```

Comment:
Don'tforgetaboutspacesbetweenelements
```suggestion
          </h1>
          
          <button
```